### PR TITLE
Support for allocating more sketches than normal

### PR DESCRIPTION
**Pull request by etwest**
We need to, for a variety of potential applications, be able to allocate more than just O(log n) sketches. This pull request gives that option to users through the `graph_configuration`.

### CODE REVIEW

File: include/graph_configuration.h [hunk 1]
```diff
@@ -30,6 +30,12 @@ class GraphConfiguration {
   // How many OMP threads each graph worker uses
   size_t _group_size = 1;
 
+  // Option to create more sketches than for standard connected components
+  // Ex factor of 1, double the sketches
+  //    factor of 0.5, 1.5 times the sketches
+  //    factor of 0, normal quantity of sketches
+  double _adtl_skts_factor = 0;
```
Comment (Victor-C-Zhang) [inline include/graph_configuration.h]: my engineer brain dislikes this very much. would be simpler for it to be just multiplicative and name it accordingly

**Review by Victor-C-Zhang — DISMISSED** [review]

**Comment by Victor-C-Zhang** [comment]
pls also document the changes you made to memory profiling/etc

**Comment by etwest** [comment]
> pls also document the changes you made to memory profiling/etc

What changes are you referring to exactly?

File: include/graph_configuration.h (hunk 1 above)
Comment (etwest) [reply include/graph_configuration.h]: Hmm okay I suppose that works at well.

Comment (etwest) [reply include/graph_configuration.h]: Done

**Review by Gillgamesh — APPROVED** [review]

**Pull request merged**